README.md

```diff
@@ -23,155 +23,80 @@
+These binaries are hosted on ASF dist subversion. The following targets are currently provided:
 
-### Install on mac
+Format: `napi-{platform}-{libc}-{arch}`
+- napi-darwin-unknown-x64.tar.gz
+- napi-linux-glibc-arm64.tar.gz
+- napi-linux-glibc-x64.tar.gz
+- napi-linux-musl-arm64.tar.gz
+- napi-linux-musl-x64.tar.gz
+- napi-win32-unknown-ia32.tar.gz
+- napi-win32-unknown-x64.tar.gz
 
-1. Install the Pulsar C++ client on mac.
+`darwin-arm64` systems are not currently supported, you can refer `How to build` to build from source.
 
-```shell
-brew install libpulsar
-```
+## How to build
 
-2. Get the installation path of libpulsar
+To build from source, you need to install the CPP client first.
 
+1. Clone repository.
 ```shell
-brew info libpulsar
+$ git clone https://github.com/apache/pulsar-client-node.git
+$ cd pulsar-client-node
 ```
 
+2. Install CPP client.
 
-2. Set the variable `PULSAR_CPP_DIR` with the `pulsar-client-cpp` path in a mac command tool.
+Select the appropriate installation method from below depending on your operating system.
 
+Install c++ client on mac
 ```shell
-# for example
-## Intel x86_64
-export PULSAR_CPP_DIR=/usr/local/Cellar/libpulsar/2.9.1_1
-
-## Apple Silicon and Homebrew since 3.0.0
-## cf. https://brew.sh/2021/02/05/homebrew-3.0.0/
-export PULSAR_CPP_DIR=/opt/homebrew/Cellar/libpulsar/2.9.1_1
-```
-
-### Install on Linux
-
-1. Require g++ and make commands to install pulsar-client
+$ pkg/mac/build-cpp-deps-lib.sh
+$ pkg/mac/build-cpp-lib.sh
 ```
-# rpm
-$ yum install gcc-c++ make
 
-# debian
-$ apt-get install g++ make
-```
-
-2. Download rpm or debian packages.
-```
-# Set the version of Pulsar C++ client to install
-$ PULSAR_CPP_CLIENT_VERSION=2.9.1
-
-# rpm
-$ wget https://archive.apache.org/dist/pulsar/pulsar-${PULSAR_CPP_CLIENT_VERSION}/RPMS/apache-pulsar-client-${PULSAR_CPP_CLIENT_VERSION}-1.x86_64.rpm
-$ wget https://archive.apache.org/dist/pulsar/pulsar-${PULSAR_CPP_CLIENT_VERSION}/RPMS/apache-pulsar-client-devel-${PULSAR_CPP_CLIENT_VERSION}-1.x86_64.rpm
-
-# debian
-$ wget https://archive.apache.org/dist/pulsar/pulsar-${PULSAR_CPP_CLIENT_VERSION}/DEB/apache-pulsar-client.deb
-$ wget https://archive.apache.org/dist/pulsar/pulsar-${PULSAR_CPP_CLIENT_VERSION}/DEB/apache-pulsar-client-dev.deb
-```
-
-3. Install the Pulsar C++ client.
+Install c++ client on Linux
 ```shell
-# rpm
-$ rpm -ivh apache-pulsar-client*.rpm
-
-# debian
-$ apt install ./apache-pulsar-client*.deb
+$ build-support/install-cpp-client.sh
 ```
 
-
-### Install pulsar-client to your project
-
+Install c++ client on Windows(Need install curl and 7z first)
 ```shell
-$ npm install pulsar-client
+$ pkg\windows\download-cpp-client.bat
 ```
 
-## Sample code
-
-Please refer to [examples](https://github.com/apache/pulsar-client-node/tree/master/examples).
-
-## How to build
-
-### Install dependent npm modules and build Pulsar client library:
+3. Build NAPI from source.
```

Comment:
I may suggest you use `### 3. Build NAPI from source` and ditto other steps, since step 2 is relatively complex.

Reply:
adopt